LeetBar/ViewModels/ViewModel.swift

```diff
@@ -63,6 +67,7 @@ final class ViewModel: ObservableObject {
 
     init(service: HTTPLeetService = LeetService()) {
         self.service = service
+        username = UserDefaults.standard.string(forKey: "username") ?? ""
```

Comment:
this is stylistic, but I tend to use `self.` in front of properties in the `init()`